csharp/extractor/Semmle.Extraction.CSharp.Standalone/BuildAnalysis.cs

```diff
@@ -350,5 +422,8 @@ public void Dispose()
         {
             packageDirectory?.Dispose();
         }
+
+        [GeneratedRegex("<PackageReference .*Include=\"(.*?)\".*/>", RegexOptions.IgnoreCase | RegexOptions.Compiled | RegexOptions.Singleline)]
+        private static partial Regex PackageReference();
```

Comment:
TIL

Reply:
Yep, this is super neat. Also, the `EnumerateMatches()` and the zero allocation `ValueMatch` is interesting.